examples/operator/README.md

```diff
@@ -12,15 +12,18 @@ kubectl apply -f operator.yaml
 # Example:
 #  images:
 #    vtctld: vitess/lite:mysql80
+#    vtadmin: vitess/vtadmin:latest
 #    vtgate: vitess/lite:mysql80
 #    vttablet: vitess/lite:mysql80
 #    vtbackup: vitess/lite:mysql80
+#    vtorc: vitess/lite:mysql80
 #    mysqld:
 #      mysql80Compatible: vitess/lite:mysql80
```

Comment:
Should we document the images to all be `:latest`?

Reply:
That comment is following a line saying that if you want to run mysql80 then change the files like the following, that is why all of them are mysql80 instead of latest.

Reply:
Oh i missed that! my bad :)